apps/web/src/app/app/[workspaceSlug]/(dashboard)/monitors/[id]/edit/page.tsx

```diff
@@ -30,6 +31,8 @@ export default async function EditPage({
 
   // default is request
   const search = searchParamsSchema.safeParse(searchParams);
+  const active =
+    search.success && search?.data?.active === "true" ? true : monitor.active;
```

Comment:
I think we can use `zod` in a way to transform `active` to a boolean.

Reply:
Hey @mxkaske ,
Can you check now?

Reply:
Done!